worlds/witness/hints.py

```diff
@@ -258,9 +258,9 @@ def make_hints(multiworld: MultiWorld, player: int, hint_amount: int):
 
     for loc, item in always_hint_pairs.items():
         if item[1]:
-            hints.append((item[0], "can be found at", loc, item[2]))
+            hints.append((item[0] + " can be found at " + loc + ".", item[2]))
```

Comment:
I think f-strings would look a lot nicer in this case. Same on line 263, 282, 284, 293 and 296
```suggestion
            hints.append((f"{item[0]} can be found at {loc}.", item[2]))
```

Reply:
God I hate f-strings so much, but you're right